ImportService/src/libs/csvParser.ts

```diff
@@ -0,0 +1,33 @@
+import { GetObjectCommandOutput } from "@aws-sdk/client-s3";
+import csv from 'csvtojson';
+import split2 from 'split2';
+import { pipeline, Transform } from "stream";
+
+let header: null | string = null;
+
+const transformStream = new Transform({
+  transform: async (chunk: Buffer, _, callback) => {
+    if (!header) {
+      header = chunk.toString();
+      callback();
+    } else {
+      const line = chunk.toString();
+      const fromCsvToArray = await csv().fromString(`${header}\n${line}`);
+      const stringified = `${JSON.stringify(fromCsvToArray[0])}\n`;
+      console.log('File line: ', stringified);
+      callback(null, stringified);
+    }
+  },
+});
```

Comment:
For the learning purposes this approach is ok, because you use Transform streams here and pipeline in the function below. But you can use `csv` function from `csvtojson` library as transform stream. So, all that you need to do is: `fileObject.Body.pipe(csv()).on('data', () => {...})`;
[Link](https://github.com/Keyang/node-csvtojson#use-stream) to the documentation.